packages/mathtype-froala3/wiris.src.js

```diff
@@ -294,19 +292,19 @@ export class FroalaIntegration extends IntegrationModel {
     // Value can be undefined.
     if (selectedImage) {
       if (($btn.parent()[0].hasAttribute('class') && $btn.parent()[0].getAttribute('class').indexOf('fr-buttons') === -1) || (selectedImage[0]
-                && ($(selectedImage[0]).hasClass(Configuration.get('imageClassName')) || $(selectedImage[0]).contents().hasClass(Configuration.get('imageClassName'))))) { // Is a MathType image.
+                && ((selectedImage[0]).classList.contains(Configuration.get('imageClassName')) || (selectedImage[0]).contents().classList.contains(Configuration.get('imageClassName'))))) { // Is a MathType image.
```

Comment:
Maybe the parentheses around `selectedImage[0]` can be dropped? And same thing with the parentheses around the whole line.
In fact, if you think it would be more readable, you could set up variables to break down the line into smaller chunks, e.g. a variable for `selectedImage[0]`, or a variable for each of the three conditionals joined by `&&`.

Reply:
settings variables for selectedImage[0] could lead to errors, since the if clause verifies the existence of that selectedImage[0], but for the parentheses parts you're right